testsuite/features/min_salt_formulas.feature

```diff
@@ -35,7 +35,7 @@ Feature: Use salt formulas
      And I select "French" in language field
      And I select "French (Canada)" in keyboard layout field
      And I click on "Save Formula"
-     Then I should see a "Formula saved!" text
+     Then I should see a text like "Formula saved*"
```

Comment:
Wildcard is no needed. The method already support it.
Please avoid non-natural language in Gherkin steps

Reply:
In fact, we don't need to use this other method, I'm not sure when that one was need.
If now we don't have "!", we only need to remove it from the step. So : Then I should see a "Formula saved" text

Reply:
Addressed.

I would stay with the `text like` method anyway.
Before the change, the message was only `Formula saved!`, now it is `Formula saved. Apply the <a href=>.......`, and in the future the message maybe will slightly change again. The relevant thing is that the message contains the `Formula saved` text without depending on the place in the sentence or anything else.

Reply:
No, "like" is for regular expressions. You are not using one. It is substring match already.

Reply:
In the end it turned out the `I should see a "..." text` implementation already works as a "partial match", so it is not needed at all the `text like` method. It is required when in the step we really have a `regex` in the text to match.
I am removing the `text like` usage then.

Reply:
Thank you very much.

Reply:
> In the end it turned out the `I should see a "..." text` implementation already works as a "partial match", so it is not needed at all the `text like` method. It is required when in the step we really have a `regex` in the text to match.
> I am removing the `text like` usage then.

Yep. That's what I tried to explain :) 
Better now. I'm not super strict on it anyway